### PR TITLE
Record component is implicitly final

### PR DESCRIPTION
**Pull request by theresa-m**
Signed-off-by: Theresa Mammarella <Theresa.T.Mammarella@ibm.com>

### CODE REVIEW

File: runtime/nls/cfre/cfrerr.nls [hunk 1]
```diff
@@ -1512,3 +1512,11 @@ J9NLS_CFR_ERR_RECORD_COMPONENT_DESCRIPTOR_NOT_UTF8.explanation=Please consult th
 J9NLS_CFR_ERR_RECORD_COMPONENT_DESCRIPTOR_NOT_UTF8.system_action=The JVM will throw a verification or classloading-related exception such as java.lang.ClassFormatError.
 J9NLS_CFR_ERR_RECORD_COMPONENT_DESCRIPTOR_NOT_UTF8.user_response=Contact the provider of the classfile for a corrected version.
 # END NON-TRANSLATABLE
+
+# RecordComponent is not translatable
+J9NLS_CFR_RECORD_CLASS_MUST_BE_FINAL=RecordComponent is implicitly final
```
Comment (DanHeidinga) [inline runtime/nls/cfre/cfrerr.nls]: Is `RecordComponent` the right word here?  The check is that the Record class itself is final (and can't be subclasses).  I would expecte `RecordComponent` to correspond to the fields in the Record.  Is that incorrect?

Comment (theresa-m) [reply runtime/nls/cfre/cfrerr.nls]: woops you're right. will update

Comment (theresa-m) [reply runtime/nls/cfre/cfrerr.nls]: done.

**Comment by theresa-m** [comment]
Port to 0.19: https://github.com/eclipse/openj9/pull/8692

**Comment by DanHeidinga** [comment]
Jenkins test sanity zlinux jdk14,jdk11

**Pull request merged**